packages/atomic/src/global/environment.ts

```diff
@@ -0,0 +1,21 @@
+import {VERSION} from '@coveo/headless';
+
+export interface AtomicEnvironment {
+  version: string;
+  headlessVersion: string;
+}
+
+function getWindow() {
+  return (window as unknown) as {Coveo: AtomicEnvironment};
+}
+
+function getAtomicEnvironment(): AtomicEnvironment {
+  return {
+    version: process.env.VERSION!,
+    headlessVersion: VERSION,
+  };
+}
+
+export default function () {
+  getWindow().Coveo = getAtomicEnvironment();
```

Comment:
This could potentially cause problem if there's multiple `Coveo` global variable attached for some reason.

Let's say a client wants to transition from JSUI to Atomic, and they have both global variable populated at the same time by including both project.

I think a solution would be to use a different variable (suggestion: `CoveoAtomic`).
We could also have some preventive code to check the existence of the variable before blindly assigning to it.